content/maxwell2_dc/fields_from_grounded_sources_dcr/electrostatic_sphere.rst

```diff
@@ -453,17 +435,8 @@ The only parameters that have changed are the radius and the conductivity of the
     electrode_spacing = 20.
     PlotOpt = 'Total'
 
-    #Initializing the figure

```

Comment:
@thast : and again here. the two_configurations_comparison code in the em_examples repository does not take ax as an input. Thanks!